amundsen_application/static/js/components/TableDetail/index.tsx

```diff
@@ -97,7 +98,9 @@ export class TableDetail extends React.Component<TableDetailProps & RouteCompone
     this.database = params ? params.db : '';
     this.schema = params ? params.schema : '';
     this.tableName = params ? params.table : '';
+
     this.displayName = params ? `${this.schema}.${this.tableName}` : '';
+    this.key = params ? `${this.database}://${this.cluster}.${this.schema}/${this.tableName}` : '';
```

Comment:
Because of the way React Router's params work, the `/` is a delimiter so I could not just put the key as a URL param. I could use `encodeURIComponent` but that results in ugly URLs and potentially broken links in Slack.

As a compromise, this is the only place where a key is jerry rigged.

Reply:
Can you add a comment and link to the line of code in the python files that this has to match? Anything that would inform someone not to change this hardcoded pattern.

Reply:
@feng-tao having trouble finding where this format is defined. Is it in databuilder or metadata?